_pydevd_sys_monitoring/_pydevd_sys_monitoring.py

```diff
@@ -1333,6 +1366,10 @@ def _jump_event(code, from_offset, to_offset):
 
     # We know the frame depth.
     frame = _getframe(1)
+
+    # Disable the next line event as we're jumping to a line. The line event will be redundant.
```

Comment:
One of the debugpy tests found that breakpoints were hitting too often. This prevents double events for the same line when jumping

Reply:
This was the test that was failing in debugpy. On this specific line:

https://github.com/microsoft/debugpy/blob/25955a05d87a6e3d7694f7479c1c17ded0f9af90/tests/debugpy/test_breakpoints.py#L152

The breakpoints would fire twice as much as expected.

Reply:
I added another test for this and fixed the special cases in the tests for 3_12 that this also fixes